common/http_test.cpp

```diff
@@ -48,6 +48,19 @@ class TestInspector {
 		return server.streams_;
 	}
 };
+
+class TestServer : public Server {
+public:
+	using Server::Server;
```

Comment:
What does this `using` do?

Reply:
It reuses the constructors from the original class, since I don't need any modifications in the sub class.

Reply:
I thought that was the default